packages/devtools_app/lib/src/app.dart

```diff
@@ -576,12 +574,15 @@ class CheckboxSetting extends StatelessWidget {
 /// Conditional screens can be added to this list, and they will automatically
 /// be shown or hidden based on the [Screen.conditionalLibrary] provided.
 List<DevToolsScreen> get defaultScreens {
+  final inspectorTreeController = InspectorTreeController();
```

Comment:
@kenzieschmoll this is an attempt to placate the duplicate args lint which I gave up on. This one should probably be reverted.

Reply:
what does the duplicate args lint prevent? two arguments aren't allowed to have the same value?

agreed this should be reverted as this is more verbose without an obvious benefit

Reply:
yeah I don't think it is useful. I do think code is generally a bit harder to read when there are multiple args passed in with the exact same expression particularly if the args aren't named but generally I think it has far too many false positives.
Reverted.